app/controllers/books_controller.rb

```diff
@@ -104,6 +114,15 @@ def show_more
     render json: books, status: 200
   end
 
+  def id_check(input_lang)
+    url_string = 'http://localhost:8080/search?q=' + input_lang + '&multilingual=true&namequerytype=whole';
+    uri = URI(url_string);
+    result = Net::HTTP.get_response(uri);
+    json_obj = JSON.parse(result.body);
+    parsed_result = (json_obj != nil) ? json_obj[0]['glottocode'] : nil
```

Comment:
we should only be setting the glottocode if we are sure there's only one matching entity, e.g. if the json result is of length 1